.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -37,24 +37,13 @@ What should happen?
 
 What happens instead?
 
-You might also want to check the last ~1k lines of
-`/opt/metasploit/apps/pro/engine/config/logs/framework.log` or
-`~/.msf4/logs/framework.log` for relevant stack traces
 
+## Additional Information
+If the issue is encountered within `msfconsole`, please run the `debug` command using the instructions below. If the issue is encountered outisde `msfconsole`, or the issue causes `msfconsole` to crash on startup, please delete this section.
 
-## System stuff
-
-### Metasploit version
-
-Get this with the `version` command in msfconsole (or `git log -1 --pretty=oneline` for a source install).
```

Comment:
For 'backwards compatibility' -- what are your thoughts on keeping this version section here:

```
### Metasploit Version

What version of Metasploit are you currently running? Get this with `msfconsole --version`

```